src/arch/arm/tlb.hh

```diff
@@ -110,6 +110,10 @@ class TLB : public BaseTLB
         using AssociativeCache<TlbEntry>::accessEntry;
         TlbEntry* accessEntry(const KeyType &key) override;
         TlbEntry* findEntry(const KeyType &key) const override;
+
+      private:
+        /** Last matched entry */
+        mutable TlbEntry *prev = nullptr;
```

Comment:
If you flush the TLB should you set this back to nullptr? Also, if you evict this entry, should you set it to nullptr?